### PR TITLE
Simplify code area categories

### PR DESCRIPTION
**Pull request by goto-bus-stop**
Changes the list to:

image

- [x] This still needs to do something with [INIT] frames, probably as a separate checkbox, building on Alan's PR that allows custom exclusion filters for d3-fg. They could be chucked under Core for now but if that PR is landing soon anyway we can also stack this PR on top of that one.

Just added it as another filter option like before, since it will keep using the `dataTree.exclude` property anyway.

### CODE REVIEW

File: visualizer/options-menu.js [hunk 1]
```diff
@@ -3,13 +3,39 @@
 const HtmlContent = require('./html-content.js')
 const d3 = require('./d3.js')
 
+function getCodeAreaDescription (area) {
```
Comment (goto-bus-stop) [inline visualizer/options-menu.js]: I moved this here to allow generating descriptions for all the dependency code areas in the future. Will prob move it to ui.js so it's in the same place as the getLabelKey() (i forget the name) one

**Comment by goto-bus-stop** [comment]
Just added 'init' as another filter option like before, since it will keep using the `dataTree.exclude` property anyway.

**Comment by goto-bus-stop** [comment]
(I have a separate PR half-ready that adds code areas for individual dependencies)

File: visualizer/ui.js [hunk 2]
```diff
@@ -331,6 +332,20 @@ class Ui extends events.EventEmitter {
...
+      'all-v8': 'The V8 engine JavaScript implementation',
+      'v8': `Operations in V8's implementation of JS. <a target="_blank" class="more-info" href="https://clinicjs.org/flame/walkthrough/controls/#v8">More info</a>`,
+      'native': `JS compiled into V8, such as prototype methods and eval. <a target="_blank" class="more-info" href="https://clinicjs.org/flame/walkthrough/controls/#native">More info</a>`,
+      'cpp': `Native C++ operations called by V8, including shared libraries. <a target="_blank" class="more-info" href="https://clinicjs.org/flame/walkthrough/controls/#cpp">More info</a>`,
+      'regexp': `The RegExp notation is shown as the function name. <a target="_blank" class="more-info" href="https://clinicjs.org/flame/walkthrough/controls/#rx">More info</a>`,
+      'init': `Module loading and other operations during initialization. Frames in this category also belong to other categories. <a target="_blank" class="more-info" href="https://clinicjs.org/flame/walkthrough/controls/#init">More info</a>`,
```
Comment (AlanSl) [inline visualizer/ui.js]: This is a bit long, and it doesn't feel like a great fit here. How about we move it to the top of the second heading section, above merged, where it'll be more intuitive that it is something that crosses other sections, and call it:

> **Init** - Show initialization operations hidden by default, like module loading.

... and then rename that section to something like "Advanced"

Comment (goto-bus-stop) [reply visualizer/ui.js]: My first thought was to add a different section but I didn't want to do that for just this item. Changing the existing one to an "advanced" section sounds good!

**Comment by AlanSl** [comment]
LGTM (pending fixing merge conflict)

**Pull request merged**